api/CSSGroupingRule.json

```diff
@@ -47,6 +47,153 @@
           "standard_track": true,
           "deprecated": false
         }
+      },
+      "cssRules": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSGroupingRule/cssRules",
+          "spec_url": "https://drafts.csswg.org/cssom/#dom-cssgroupingrule-cssrules",
+          "support": {
+            "chrome": {
+              "version_added": "45"
+            },
+            "chrome_android": {
+              "version_added": "45"
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "firefox": {
+              "version_added": "20"
+            },
+            "firefox_android": {
+              "version_added": "20"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "32"
+            },
+            "opera_android": {
+              "version_added": "32"
+            },
+            "safari": {
+              "version_added": false
```

Comment:
Safari does have `CSSGroupingRule`, but the parent feature is wrong too. I've filed https://github.com/mdn/browser-compat-data/issues/11644.